docs/gpa.md

```diff
@@ -194,28 +194,26 @@ The information included in the notice of intended procurement covers all the in
 
 </td>
       </tr>
-<!--
       <tr>
         <td>VII:4</td>
         <td>Procuring entities are encouraged to publish in the appropriate paper or electronic medium listed in Appendix III as early as possible in each fiscal year a notice regarding their future procurement plans (hereinafter referred to as "notice of planned procurement"). The notice of planned procurement should include the subject-matter of the procurement and the planned date of the publication of the notice of intended procurement.</td>
```

Comment:
I think we need to say something about the `tag` field ('planning') and the `tender.status` field ('planning').

Reply:
That makes me think that we don't  have "Create a release" guidance in the GPA profile. There is no reference to `ocid` nor release `id`. The concept of release isn't mentioned. Mentioning the addition of a `tag` implies that distinct releases may be published.

For now I'll just mention "Add tag 'planning'", as if all the data was published in the same release. In practice, I suppose it may or may not be the same.

This issue highlights the difficulty to produce extensive guidance when the source standard is not structured.

Reply:
Done